### ISSUE TITLE
XCP-D failed: '>' not supported between instances of 'NoneType' and 'float'

### ISSUE THREAD

**Issue by chenfei-ye** [PennLINC/xcp_d#1127]
Hi, 

xcp-d docker container 0.7.0 works well for fmriprep/hcp input, but failed for version > 0.7.0 (tested on 0.7.1rc3 and 0.7.1rc5), with the following prompt:

240405-06:05:37,319 nipype.workflow CRITICAL:
         XCP-D failed: '>' not supported between instances of 'NoneType' and 'float'

Here is my command for hcp input:
```
docker run --rm -it -v /data3/HCP/input:/fmriprep:ro  \
    -v /data3/HCP/wkdir:/work:rw  -v /data3/HCP/xcp_d:/out:rw \
    -v /data/freesurfer/:/fs_license pennlinc/xcp_d:0.7.1rc5 /fmriprep /out participant \
    --cifti --despike --head_radius auto -w /work --input-type hcp --nthreads 20 \
    --mem_gb 10 --fd-thresh 0 --nuisance-regressors 36P \
    --fs-license-file /fs_license/freesurfer_license.txt --min-coverage 0.2 \
    --stop-on-first-crash --atlases 4S156Parcels 4S256Parcels 4S456Parcels HCP
```

and my command for fmriprep:
```
docker run --rm -it -v /data3/test/fmriprep:/fmriprep:ro  \
    -v /data1/hit_data/PD_tt/bids/derivatives/wkdir:/work:rw  \
    -v /data1/hit_data/PD_tt/bids/derivatives/xcp_d:/out:rw \
    -v /data/freesurfer/:/fs_license pennlinc/xcp_d:0.7.1rc5 \
    /fmriprep /out participant --despike --head_radius auto -w /work  \
    --nthreads 10 --mem_gb 10 --fd-thresh 0 --nuisance-regressors 36P \
    --fs-license-file /fs_license/freesurfer_license.txt --min-coverage 0.2 \
    --stop-on-first-crash --atlases 4S156Parcels 4S256Parcels 4S456Parcels HCP \
    --participant-label HC001 
```

Any idea?

**Comment by tsalo** [comment]
This looks like a bug reported in #1123. I believe I fixed it in 0.7.1rc5, and I'm having trouble reproducing it locally using 0.7.1rc5 on some data I have on hand. Are you 100% sure you used 0.7.1rc5 and not 0.7.1rc4?

**Comment by chenfei-ye** [comment]
> This looks like a bug reported in #1123. I believe I fixed it in 0.7.1rc5, and I'm having trouble reproducing it locally using 0.7.1rc5 on some data I have on hand. Are you 100% sure you used 0.7.1rc5 and not 0.7.1rc4?

That's my bad. this issue occurs for pennlinc/xcp_d:latest, and I was thinking it's the same with pennlinc/xcp_d:0.7.1rc4. 
In my test, 0.7.1rc5 works well. Thank you for your reply.
